website/content/en/docs/main/concepts/policies.md

```diff
@@ -7,14 +7,14 @@ weight: 40
 Access policies allow users and administrators fine-grained control over
  which client workloads may access which service. This is an important security
  mechanism for applying [micro-segmentation][], which is a basic requirement of [zero-trust][]
- systems. Another zero-trust principle, "Deny by default / Allow by exception", is also
- addressed by ClusterLink's access policies: a connection without an explicit policy allowing it,
+ systems. Another zero-trust principle, "Deny by default / Allow by exception," is also
+ addressed by ClusterLink's access policies: a connection without an explicit policy allowing it
  will be dropped. Access policies can also be used for enforcing corporate security rules,
  as well as segmenting the fabric into trust zones.
 
 ClusterLink's access policies are based on attributes that are attached to
  [peers][], [services][] and client workloads.
- Each attribute is a key:value pair, similar to how [labels][]
+ Each attribute is a key-value pair, similar to how [labels][]
```

Comment:
I believe it should stay with :

Reply:
Thanks for your comment, Michal! Trying to guess your reasons - is that how it is coded, so you think it would be clearer to the user that way? (Let me know if I'm missing something). If we wanted to call key-value a compound adjective describing the noun "pair," key-value would work from an English standpoint. It's used that way (with a hyphen) for example on this website. https://aws.amazon.com/nosql/key-value/#:~:text=A%20key%2Dvalue%20database%20is,serves%20as%20a%20unique%20identifier.
Please let me know what you think after reviewing this info, or feel free to let me know what I may be missing. Thanks!

Reply:
also here (with the word pair) https://experienceleague.adobe.com/en/docs/audience-manager/user-guide/reference/key-value-pairs-explained#:~:text=A%20key%2Dvalue%20pair%20consists,color%20%3D%20green

Reply:
Official [k8s documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) uses `Key/Value`